### PR TITLE
Accessibility docs page tweaks

### PR DESCRIPTION
**Pull request by junpataleta**
- Make the wording more generic and not just pertaining to a single occasion of the accessibility audit.
- Remove the accreditation badges (especially since they are currently expired) and replace them with links to the accreditation pages.
- List down the dates when WCAG 2.1 Level AA accreditation was issued on the given Moodle versions on a table.

<a href="https://gitpod.io/#https://github.com/moodle/devdocs/pull/513"></a>

### CODE REVIEW

**Review by andrewnicols — APPROVED** [review]
Is it worth adding a link to the relevant issue for each of the WCAG reviews?

Also is it worth noting that the accreditation does expire, but that we still achieved compliance in the first place and that we do not recertify versions after their initial certification. That seems to be the number two recurring question after "When is Version X going to be certified?"

I'll let you self-merge this if you want to now, or you can add a patch with any of the above here or in a new issue. Up to you.

**Comment by junpataleta** [comment]
Thanks, @andrewnicols. I added a [fixup commit](https://github.com/moodle/devdocs/pull/513/commits/a5d2bf96859d42c421b2115fb2b1d58f32e3331b) where:
- The epic links for the accessibility audits are added on the table for Moodle LMS
- I added a note about the expiry of the accreditation. Though I am not sure if I worded it appropriately or whether it can be further improved/made clear. Please feel free to suggest improvements. 

Marie will also be checking this out sometime soon and am looking for her feedback too.

Cheers!

File: general/development/policies/accessibility.md
Comment (andrewnicols) [inline general/development/policies/accessibility.md:21]: I'd suggest something like t his maybe:

```suggestion
Accreditations expire after one year, but this does not mean that the accreditation is invalid or that the Moodle version is no longer accessible. An expired accreditation just means that the process for re-accrediting the latest supported version or versions of Moodle is currently in progress.
```

Remove the distinction between "in progress" and "pending", and put it into a slightly more complete and full sentence.

Comment (junpataleta) [reply general/development/policies/accessibility.md]: Thanks, Andrew! I have applied your suggestion, amended the fixup commit and squashed it to the original commit.

**Pull request merged**
